README.md

```diff
@@ -2,11 +2,15 @@
 
 Dynamic Visualization tool for machine learning models that displays the models
 
+![](gif_demo.gif)
+
 ## For Users
 ### Graph Visualization Usage Documentation
 
 #### Usage
-`python -m graph_visualization.plot_graph`<br/>`--pb_graph_path=`_`path_to_protobuf/protobuf.`_`pb`
+```pip install model-vis<br/>```
+
+```python -m graph_visualization.plot_graph```<br/>```--pb_graph_path=```_```path_to_protobuf/protobuf.```_```pb```
```

Comment:
can you change this so it looks like, this way you can copy it into your terminal

```
python -m graph_visualization.plot_graph \
--pb_graph_path=path_to_protobuf/protobuf.pb
```

Reply:
like you don't need line breaks, just do the whole thing inside ```